README.md

```diff
@@ -257,11 +261,15 @@ Services:
  - [Mongo Seeding](https://github.com/pkosiec/mongo-seeding) - Node.js library, CLI and Docker image for populating databases using JS and JSON files
  - [Mongoeye](https://github.com/mongoeye/mongoeye) - Schema and data analyzer: explore data in your collections
  - [Variety](https://github.com/variety/variety) - Schema analyzer: see what fields are in your collection and what's their content
+ - [MongoDB .NET Analyzer](https://www.mongodb.com/docs/mongodb-analyzer/current/)
+ - [MongoDB VS Code Extension](https://www.mongodb.com/docs/mongodb-vscode/)
 
 Services:
+ - [MongoDB Atlas App Services](https://www.mongodb.com/atlas/app-services) - MongoDB Inc. solution to run code without the operational overhead
 
- - [MongoDB Realm](https://www.mongodb.com/realm) - MongoDB Inc. development services and SDK
-
+### Mobile
+ - [MongoDB Realm](https://www.mongodb.com/realm) - MongoDB Inc. solution for mobile data sync
```

Comment:
Thanks for the description update. I'm not sure it's worth a new section thought, and it should be under a `Services:`